challenges/functions.js

```diff
@@ -7,24 +7,46 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume(x, y, cb) {
+  return cb(x, y);
+}
+
+
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+  function add(x, y) {
+    return x+y;
+  }
+
+  function multiply(x, y) {
+    return x*y;
+  }
+
+  function greeting(first, last) {
+    return 'Hello' + ', ' + first + ' ' + last + ', ' + 'nice to meet you!' }
```

Comment:
ES6 syntax preferred! Side note: if using this style you could do "Hello, " + first + '" " + last " " nice to meet you."